Packs/GoogleSafeBrowsing/Integrations/GoogleSafeBrowsingV2/GoogleSafeBrowsingV2.yml

```diff
@@ -0,0 +1,113 @@
+category: Data Enrichment & Threat Intelligence
+commonfields:
+  id: Google Safe Browsing v2
+  version: -1
+configuration:
+- defaultvalue: https://safebrowsing.googleapis.com/v4/threatMatches:find
```

Comment:
can you explain the `threatMatches:find` suffix?

Reply:
This is the URL - 
![image](https://user-images.githubusercontent.com/57212002/116363630-014f9a00-a80c-11eb-8d35-9b3b60137a42.png)